Sources/RTMP/RTMPTSocket.swift

```diff
@@ -89,6 +89,9 @@ final class RTMPTSocket: NSObject, RTMPSocketCompatible {
         for chunk in chunks {
             bytes.append(contentsOf: chunk)
         }
+        if (locked != nil) {
+            OSAtomicAnd32Barrier(0, locked!)
```

Comment:
@shogo4405 same as in `NetSocket.swift` unlocks video encoder not sure this is the right place to put it though